rapid_response_xblock/logger.py

```diff
@@ -2,18 +2,74 @@
 Capture events
 """
 from __future__ import absolute_import
+import logging
 
+from django.db import transaction
+from opaque_keys.edx.keys import UsageKey
+from opaque_keys.edx.locator import CourseLocator
 from track.backends import BaseBackend
 
+from rapid_response_xblock.models import ProblemCheckRapidResponse
+
+
+log = logging.getLogger(__name__)
+
 
 class LoggerBackend(BaseBackend):
     """
     Record events emitted by blocks.
     See TRACKING_BACKENDS for the configuration for this logger.
     For more information about events see:
 
-    http://edx.readthedocs.io/projects/devdata/en/stable/internal_data_formats/tracking_logs.html
+    http://edx.readthedocs.io/projects/devdata/en/stable/
+    internal_data_formats/tracking_logs.html
     """
 
     def send(self, event):
-        pass
+        # TODO: feature flag whitelisting valid problems
```

Comment:
I'm confused. I thought we were setting up some listener to respond to a django signal that gets emitted when a problem is answered. Is there a reason that wouldn't work or isn't optimal? Also, it's confusing that this logic is kept in a method of a class called `LoggerBackend`. When I see "Logger", I'm thinking that this class is just responsible for emitting log messages, not implementing critical functionality

Reply:
The events aren't django signals, the event dispatch infrastructure is specific to edX. `LoggerBackend` comes from the edX implementation but I agree it's confusing, I'll rename it